stable/traefik/values.yaml

```diff
@@ -388,6 +388,11 @@ deployment:
     # httpPort: 80
     # httpsPort: 443
     # dashboardPort: 8080
+  containerPort:
```

Comment:
Can you configure these to be optional? (commented out by default).

Reply:
I would prefer the approach suggested in this PR:
* DRY: Have the default ports in values.yaml in one place instead of having a lot of `default` statements
* I would have to add a key to `.Values.deployment.containerPort` as otherwise the `default` will crash into a nil pointer. Having `enabled: true` makes no sense for containerPort so this will be ugly

Reply:
I feel you however,

> Have the default ports in values.yaml in one place instead of having a lot of default statements
These ports are already hard coded in the templates, so this isn't as much of an issue

> I would have to add a key to .Values.deployment.containerPort as otherwise the default will crash into a nil pointer. Having enabled: true makes no sense for containerPort so this will be ugly
You have to check if the value exists before using it. This is used many times in the chart for optional values (https://github.com/helm/charts/blob/496c59c31c57eb0c9c15bb6673bf29408d149ef7/stable/traefik/templates/configmap.yaml#L72)

The major reason for this is backwards compatibility. If we merge this, every single user that wants to update the chart has to add these values in. I don't think that is ideal at this stage of the game.

Reply:
Are you sure? The values.yaml will always be used and values from `--set=` or `-f moreValues.yaml` will only be overriding them.

However the reason why I added this was that low ports are not allowed in my Kubernetes setup. I could change the hardcoded ports to something > 1024 and it should work too.